pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -2,26 +2,28 @@
 
 ### Deposit to Subaccount
 
+The below cmd allows depositing funds into a subaccount from a sender. This can be a self deposit with the sender and recipient address being the same
```

Comment:
```suggestion
The below cmd allows depositing funds into a subaccount (`x/subaccounts`) from a sender's `x/bank` account. 

Typically, this cmd is used to self deposit, which can be done by setting the sender and recipient address to be the same address.
```

Reply:
"Typically, this cmd is used to self deposit, which can be done by setting the sender and recipient address to be the same address"

Is this true? Seems like it doesn't actually have to be

Reply:
Done